src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -99,7 +99,15 @@
      */
     @Test
     public void handlesSocket() throws IOException {
-        final Socket socket = createMockSocket();
+        final Socket socket = createMockSocket(
```

Comment:
@longtimeago absolutely no. I asked about it in #539 but decided it will be faster to create PR with anything. Could you please explain what should I do here?